Input/Input.js

```diff
@@ -26,7 +26,7 @@ import {DEFAULT_LENGTH, calcAriaLabel, convertToPasswordFormat, extractInputFiel
 import componentCss from './Input.module.less';
 
 const prepareInputEventPayload = ev => ({value: ev.target.value});
-const isPasswordType = type => type === 'password' || type === 'passwordnumber';
+const isPasswordType = type => type.includes('password');
```

Comment:
A bug is found by unit-test.

```suggestion
const isPasswordType = type => type && type.includes('password');
```

Reply:
Fixed. Thanks for finding!